cluster-scope/base/core/namespaces/peribolos-as-a-service/namespace.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: peribolos-as-a-service
+    annotations:
+        openshift.io/display-name: Peribolos Service
+        openshift.io/requester: peribolos
```

Comment:
@oindrillac could you please add [these annotations](https://github.com/operate-first/apps/blob/master/cluster-scope/base/core/namespaces/aicoe-meteor/namespace.yaml#L8-L10) to this namespace yaml? 
This will make this project available on the operate-first website workload overview

Reply:
@4n4nd sure, added!